flixel/FlxCamera.hx

```diff
@@ -1190,6 +1201,16 @@ class FlxCamera extends FlxBasic
 	}
 	
 	/**
+	 * Sets the filter array to be applied to the camera.
+	 * 
+	 * @param	filters
+	 */
+	public function setFilters(filters:Array<BitmapFilter>):Void
```

Comment:
What's the advantage of having a setter function? Why not just make `_filters` a `public var filters`?